src/mixins/amalgamation-mixin.ts

```diff
@@ -55,42 +55,48 @@ export default class AmalgamationMixin extends Vue {
   //   }
   // ]
 
+  /**
+   * Get the amalgamating business information, mailing address, and email if in LEAR.
+   * Otherwise, return error.
+   */
+  async fetchTingInfo (item: any): Promise<AmalgamatingBusinessIF> {
+    let business = null
```

Comment:
@severinbeauvais Sev, once you merge your changes, I'll change this to whatever you decide upon in your PR.